data/Glitched World/Overworld.json

```diff
@@ -609,17 +609,36 @@
             "Haunted Wasteland": "True"
         }
     },
+    # A collection delay hookshot jump can provide access to the arch, but
+    # collection delay is never in logic due to unrepeatability
     {
         "region_name": "Desert Colossus",
         "scene": "Desert Colossus",
         "hint": "DESERT_COLOSSUS",
         "time_passes": true,
-        "locations": {
+        "events": {
+            "Adult Spirit Hover Is Repeatable": "
```

Comment:
```suggestion
            # This is needed for ER reasons where chus may be locked to one specific age
            # And when both ages must pass through spirit to reach ToT
            "Adult Spirit Hover Is Repeatable": "
```

Reply:
Not happy with my wording on this. Might come back to it later.

Reply:
I added "Used in Spirit logic, checks that adult can return to the hands after either age reaches them once from here"